advanced/advanced_numpy/examples/myobject_test.py

```diff
@@ -4,6 +4,7 @@
 
 Playing with the module defined in myobject.c
 """
+from __future__ import print_function
```

Comment:
It would be good if you add a line of comment above such a line to explain what this does. We have to keep in mind that this is a teaching material.